drivers/video/video_esp32_dvp.c

```diff
@@ -351,6 +426,13 @@ static int video_esp32_init(const struct device *dev)
 		return -ENODEV;
 	}
 
+	#ifdef CONFIG_VIDEO_ESP32_VSYNC_INTERRUPT
+	if (!video_esp32_enable_interrupt(dev)) {
```

Comment:
Why not capture the result of this function and return it instead of EIO?

Reply:
Why would this be better?